the-collab-lab/website-og
# Add mentors and participants to About page

## Description

We've decided to add an About page to the site that includes all Collab Lab mentors and participants. The idea is to help people feel comfortable trusting Collab Lab by showing the wonderful people who’ve taken part and mentored.

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|   ✓ | :sparkles: New feature     |
|   ✓ | :hammer: Refactoring       |


## Before / After

**Before:** No specific details about the wonderful folks who are involved with Collab Lab. 

**After:** A responsive about page that features our participants and mentors, plus some additional details about the history of The Collab Lab. 

## Testing Steps / QA Criteria

* Visit `http://localhost:8000/about-us/`
* Check out UI in all screensizes to make sure nothing terribly wonky is happening. 
* Look for typos and other funny business
* Review for accessibility

## Screenshots

![full screen](https://zappy.zapier.com/11c3ff9e00dc34dffc9fafb0cec1bf7a.png)
![medium screen](https://zappy.zapier.com/edd7ea15c0c19658644162a72a35af4f.png)
![small screen](https://zappy.zapier.com/51a10b56fb12602846c1477cb9082f82.png)
